rules_python_gapic/pytest.py

```diff
@@ -0,0 +1,11 @@
+import sys
+import pytest
+import os
+
+
+if __name__ == '__main__':
+    sys.exit(pytest.main([
+        '--disable-pytest-warnings',
```

Comment:
Should we keep warnings?

Reply:
It emits a lot of garbage warnings, contaminating the output and those warnings seem to be more related to the execution environment instead of the tests themselves, so I decided to turn them off. It won't affect execution much, but it will make our log even longer (the current full build log accross whole googleapis is multple megabytes already)